tb_plugin/README.md

```diff
@@ -184,6 +192,8 @@ Host Self Duration: The accumulated time spent on Host, not including this opera
 
 Host Total Duration: The accumulated time spent on Host, including this operator’s child operators.
 
+CallStack: All call stacks of this operator. The TensorBoard has integrated to VSCode, if you launch TensorBoard in VSCode, clicking this CallStack will forward to corresponding line of source code.
```

Comment:
Lack of a picture showing the expanding callstacks.
And a picture showing VSCode navigation to code line.

Reply:
done , thanks